src/components/RecruiteeJobsList.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
+import { Headline, Lead } from 'components';
+
+const RecruiteeJobsList = () => {
+  useEffect(() => {
+    const rtscript = document.createElement('script'); 
+    rtscript.type = 'text/javascript';  
+    rtscript.onload = function() { 
+      new window.RTWidget({ 
+        companies: [61772],
+        detailsMode: 'popup',
+        language: 'en',
+        departmentsFilter: [],
+        themeVars: {
+          primary: '#e8590c',
+          secondary: '#222',
+          text: '#5c6f78',
+          textDark: '#37474f',
+          fontFamily: '"Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;',
+          baseFontSize: '16px',
+        },
+        flags: {
+          showLocation: true,
+          showCountry: false,
+          showCity: false,
+          groupByLocation: false,
+          groupByDepartment: false,
+          groupByCompany: false,
+        }
+      });
+    }
+    rtscript.src = 'https://d10zminp1cyta8.cloudfront.net/widget.js';
+    document.body.appendChild(rtscript);
+
+    return () => {
+      document.body.removeChild(rtscript);
+    }
+  });
```

Comment:
Add deps array
```suggestion
  }, []);
```

Reply:
Interesting. I kinda understand what that does when there are items in the array. What's the difference between an empty array and none at all?

Reply:
none at all will rerun the function on every render. Empty on the first render. It doesn't make a difference on this site because the page will be rendered most likely once (Gatsby prerenders and it'll be HTML already), but on other sites it may cause an infinite loop or dupe resources etc.

Reply:
Appreciate the explanation!